src/blog/2023/07/images-in-node-red-dashboards.md

```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
 
 - displaying physical tools which should be used to resolve a problem.
 
+## Prerequisites
+
+Before proceeding, ensure that you have the following custom nodes installed, as they will be used throughout this guide:
+
+- [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) - A Node-RED node to perform basic mathematical calculations.
+- [node-red-contrib-image-output](https://flows.nodered.org/node/node-red-contrib-image-output) - A simple way to preview and examine images in your flows.
+- [node-red-contrib-os](https://flows.nodered.org/node/node-red-contrib-os) - Nodes for obtaining system information like CPU usage.
+- [node-red-contrib-string](https://flows.nodered.org/node/node-red-contrib-string) - A string manipulation node based on the lightweight stringjs library.
+- [@flowfuse/node-red-dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) - A set of dashboard nodes for Node-RED.
+- [node-red-node-base64](https://flows.nodered.org/node/node-red-node-base64) - A Node-RED node to encode and decode data to and from base64.
+
+## Easily Add Images to Node-RED Dashboards with FlowFuse’s Static Asset Service
+
+[FlowFuse's static assets](https://flowforge.com/docs/user/static-asset-service/) service provides a simple way to manage images and other assets in Node-RED. Follow these steps to quickly add images to your Node-RED dashboard.
+
+### Steps to Add Images Using the Static Asset Service:
+
+##### 1. Access the Static Assets Service
+- Log into your FlowFuse account, navigate to your **Node-RED instance**, and click on the **Static Assets Service** tab.
+
+##### 2. Create a New Folder (Optional)
+- Click New Folder to organize your assets, provide a folder name, and **confirm**.
```

Comment:
```suggestion
- Click the **New Folder** button to create a folder that will help you organize your assets. Provide a folder name and **confirm** the creation.
```

Reply:
@sumitshinde-84 Your suggestion is correct. I will commit it